docs/includes/topic_new-supported-features.rst

```diff
@@ -0,0 +1,16 @@
+New Supported Features
+======================
+
+Support for the following F5® features (also referred to as 'configurable features') is introduced in v |release|:
+
+* BIG-IP® device clusters (:ref:`f5_ha_type <HA mode>`)
+* `SSL offloading <https://f5.com/glossary/ssl-offloading>`_
+
+
+Support for the following OpenStack |openstack| `features <http://docs.openstack.org/releasenotes/neutron-lbaas/unreleased.html#new-features>`_ is introduced in v |release|:
+
+* Listener :ref:`TERMINATED_HTTPS <Certificate Manager>` protocol
+* Shared pools [#]_
+
+
+.. [#] Multiple listeners can be associated with a single pool.
```

Comment:
All set! Link for review: http://lbaasv2-test.readthedocs.io/en/feature-userguide-new/release_notes.html#new-supported-features